npm-debug.log

```diff
@@ -0,0 +1,95 @@
+0 info it worked if it ends with ok
```

Comment:
You might have committed this file by mistake?

Reply:
that is weird, did not appear when I did `git diff`, besides, I did not create it manually, so it is created automatically.. but it is not ignored in `.gitignore`..

Reply:
can you help me on this? I did:

``` bash
git rm --cached npm-debug.log
git add -A
git commit -m 'remove file commited by mistake'
git push
```
But did not worked as expected :thinking:

Reply:
Which part did not work?

Reply:
I don't see my commit here :thinking:... I pushed it

Reply:
Are you sure `git push` did not give an error?